pages/background.md

```diff
@@ -1,25 +1,49 @@
+[Home](https://michaelrappa.github.io/usa-migration/) |
+[Analysis](https://michaelrappa.github.io/usa-migration/pages/analysis.html) |
+[Background](https://michaelrappa.github.io/usa-migration/pages/background.html) |
+[Reflections](https://michaelrappa.github.io/usa-migration/pages/reflections.html) |
+[Workflow](https://michaelrappa.github.io/usa-migration/pages/workflow.html) |
+[Credits](https://michaelrappa.github.io/usa-migration/pages/credits.html) |
+[Bibliography](https://michaelrappa.github.io/usa-migration/pages/bibliography.html) |
+[Sources](https://michaelrappa.github.io/usa-migration/pages/sources.html)
+
+
+
 # Project Background
 
+![Immigration]({{ site.baseurl }}/assets/images/Opendomainimmigrationpic.jpg)
+
+
 ---
+> "The United States is the only major world power
+ today that owes its existence primarily to the phenomenon of
+ immigration" - Ernest Rubin
+ 
+Our project will look at immigration into the United States. We will look at the major countries that have come to the United States and use data visualization to analyze trends in relation to major world events and immigration laws. The project will look at immigration laws and world events to see patterns or trends in the data and hope to come to conclusions with it. We will be using data found from credible sources and plugging them into Tablaeu to create the visualizations. 
 
-Put a general discussion of your topic here. Don't get into the historiography or scholarship here. Just introduce to your reader what your topic is. What is the 'problem', the 'question', and/or the 'argument'.
 
-YOU SHOULD BE LIBERAL AND INCLUDE ANY AND ALL LINKS WHEN RELEVANT
 
 ---
 
-## Historical Background
+## Historical and Digital Background
 
-This is where your historiography goes. You can discuss the historiography of specific times and places, but you can also discuss any theoretical literature you think is relevant.
+Immigration to the United States has been a staple to our country’s growth both culturally and economically since the nations birth. Historically, there has been multiple causes for immigration as Quinn states. He states that immigrants look to escape conflict, healthcare, escape poverty "The American Dream", provide opportunities toward children, and jobs. The past decade has put immigration up in political strife where the nation is divided on the current situation. Currently as it stands, Mexico is on top of the debate. There is a lot of push and pull for reasons on why Mexicans seek to immigrate here.  Aguila points out three categories for factors that cause immigration flows that can be used as a model for the rest of the countries in this project as well. These categories are:
```

Comment:
```suggestion
Immigration to the United States has been a staple to our country’s growth both culturally and economically since the nations birth. Historically, there has been multiple causes for immigration as Quinn states that immigrants look to escape conflict, healthcare,  poverty, and that 'The American Dream', provides opportunities toward children, and jobs (5). The past decade has put immigration up in political strife where the nation is divided on the current situation. Currently as it stands, Mexico is on top of the debate. There is a lot of push and pull for reasons on why Mexicans seek to immigrate here.  Aguila points out three categories for factors that cause immigration flows that can be used as a model for the rest of the countries in this project as well (11). These categories are:
```